lib/pbench/server/indexer.py

```diff
@@ -1774,16 +1774,16 @@ def _handle_iteration(self, iter_data, iter_name, iter_number, result_json):
             # FIXME: this is a "traffigen" specific field; we take care of
             # this here because we don't have a mechanism for source-data-
             # specific transformations external to the code.
-            bm_md["trafficgen_uid_tmpl"] = bm_md["trafficgen_uid"]
+            bm_md["uid_tmpl"] = bm_md["trafficgen_uid"]
```

Comment:
Does this mean we can remove the `trafficgen_uid` field from json?

Reply:
@portante can we get an official experienced interpretation here? This seems to be what you were driving for initially with your comment during the presentation; but how sure are we this won't break anyone down the line?

Reply:
I think we should address this kind of change as a dedicated PR.  @npalaska, can you file an issue to track this?

Reply:
Will do

Reply:
filed an issue to track https://github.com/distributed-system-analysis/pbench/issues/1852